### DISCUSSION TITLE
Feature for runners

### DISCUSSION THREAD

**Discussion by SlippyJimmy** [time4tea/gopro-dashboard-overlay#158] [Ideas]
I hope the author can add the following functions, which are very useful for runners.
1. Time since start：Similar to odo(Distance since start).This can reflect the time since the start of exercise.
2. Max distance：This function is great when paired with the next function
3. Zone bar's Max & Min Values can be set to floating point number,and the effect (from Telemetry Overlay) shown below can be achieved（min value is zero,and max is max distance)
<img width="525" alt="image" src="https://github.com/time4tea/gopro-dashboard-overlay/assets/45062658/4ca2730a-1f18-4a34-86be-a2646ba92d20">

4. Cairo Circuit Map:I recorded multiple videos during one exercise, and they were discontinuous. When I use the `Creating Videos using only GPX File` function to generate overlay based on the time of each video, the map is also discontinuous and only displays the motion trajectory corresponding to this video. How can we ensure that the motion trajectory in the generated floating layer video is the motion trajectory of the entire motion? (That is, the motion trajectory corresponding to the entire gpx file)

**Comment by SlippyJimmy** [comment]
Add: Cairo Circuit Map XML code
<img width="1227" alt="image" src="https://github.com/time4tea/gopro-dashboard-overlay/assets/45062658/3d5b382e-db99-432a-a69b-bd980a41f09c">

**Comment by time4tea** [comment]
Thanks for these suggestions - its always great to learn a bit more about what people want from the software...

**Comment by time4tea** [comment]
With regard to (4) - currently the GPX data is clipped to the dates of the video. It has always worked this way, although perhaps it should work differently.. thanks for raising this - I'll have a think about it!

**Reply by SlippyJimmy** [reply]
Maybe I didn't express it very clearly. Let me re-explain the application scenario: For example, I cycled or ran for 3 hours, and the GPX file also recorded the movement trajectory for three hours. However, due to the limited power of the shooting equipment, I only recorded 30 minutes of video in segments during the exercise. 
1.  If you use the `--use-gpx-only` parameter to generate a video floating layer based on the GPX file, the displayed motion trajectory will be complete, but the rendering time will be long, and many clips in this floating layer video do not correspond to the captured images and need to be removed in post-editing. 
2.  If you use the `--video-time-start` parameter to generate a video floating layer based on (1), the floating layer and the picture correspond to each other. But there is a problem. The motion trajectory in the floating layer is not a complete motion trajectory. It only displays the corresponding motion trajectory for each video period. It is recommended to add custom parameters under the `--video-time-start` parameter to generate a complete motion trajectory.

**Reply by SlippyJimmy** [reply]
This 30 minutes of video is discontinuous and divided into multiple recordings.

**Comment by fid0did0** [comment]
Nice Idea, also a "reversed odo" (Distance to end) defined as max_distance-odo shall be nice to have.
